pkg/sdk/tags_validations.go

```diff
@@ -144,3 +141,25 @@ func (opts *undropTagOptions) validate() error {
 	}
 	return errors.Join(errs...)
 }
+
+func (opts *setTagOptions) validate() error {
+	if opts == nil {
+		return errors.Join(ErrNilOptions)
+	}
+	var errs []error
```

Comment:
do we want to validate what objectTypes can be called with this function or we let it fail on Snowflake level?

Reply:
i think its fine to let it fail on Snowflake level.